lib/cisco_node_utils/bgp_neighbor_af.rb

```diff
@@ -544,23 +557,49 @@ def default_route_reflector_client
     end
 
     # -----------------------
-    # <state> send-community [ both | extended | standard ]
-    # NOTE: 'standard' is default but does not nvgen on some platforms
-    # Returns: none, both, extended, or standard
+    # Nexus and XR implementations of send community differ enough that it makes
+    # sense to split up the individual methods to support them
     def send_community
       val = config_get('bgp_neighbor_af', 'send_community', @get_args)
       return default_send_community if val.nil?
+      platform == :nexus ? send_comm_nexus_get(val) : send_comm_ios_xr_get(val)
+    end
+
+    # Nexus: <state> send-community [ both | extended | standard ]
+    #  NOTE: 'standard' is default but does not nvgen on some platforms
+    #  Returns: none, both, extended, or standard
+    def send_comm_nexus_get(val)
```

Comment:
I shortened it so I could get the terniary on one line. You ok with:

      platform == :nexus ? send_community_nexus(val) : send_community_xr(val)
?